apps/core/scenes/results/Results.js

```diff
@@ -165,6 +165,7 @@ class Results extends React.Component<Props> {
       },
     };
     const desktopLayout = this.props.layout >= LAYOUT.desktop;
+    const isMobile = this.props.layout < LAYOUT.largeMobile;
```

Comment:
Just one last small thing and I believe we can merge this one after that.
Can we name this `mobileLayout` to keep it consistent with const line above? Or maybe use `desktopLayout` directly and make whole page scrollable even for tablet-like resolutions.